### ISSUE TITLE
Firecompass disables the live edit function of firebug on  inline/internal styled elements

### ISSUE THREAD

**Issue by dougcassidy** [is-already-taken/firecompass#9]
Here's a pen: http://codepen.io/JFrankParnell/pen/zxbZZR

```
<link rel="stylesheet" type="text/css" href="http://dougcassidy.com/test.css">
<style>
  span{ color:red; }
</style>
<span>cant change me</span>
<div>you can change me</div>
```

If I inspect the <span> with firebug, using the target button, I can see that my 
style is color: red, obviously.  Normally, I could change color, in 
firebug to blue or whatever. Normally, I could click the disable thingy,
 in front of the style, to grey it out and it should go back to default 
color- no effect.  I cant add any other styles to it either, by double clicking and
 adding a style.   I can change or disable the other element, in 
firebug, no prob. 

Latest FF, FB, FC.  Win 7.  I narrowed this down to firecompass, in my plugins.

**Comment by ghost** [comment]
Confirmed. Also, you _can_ edit the red text at the link above if you click the drop-down arrow on the Style tab and select 'Filename and Line number: of .css file' instead of 'Filename and Line number: of .scss file'...

**Comment by dougcassidy** [comment]
Yes, and I found that when youre on 'Filename and Line number: of .css file'  you can see your .scss file and line# by hovering over the css filename, in firebug.
